docs/admin/install/steps/install-configure.rst

```diff
@@ -47,7 +47,7 @@ Configuring Weblate
 
          ~/weblate-env/lib/python3.7/site-packages/weblate/examples/celery start
 
-#. Start the development server (see :ref:`server` for production setup):
+#. Start the development server:
```

Comment:
What is the reason for removal here?

Reply:
It occurs below

Reply:
Yes, it occurs, but it is important to point people to the production setup here.

Reply:
It's already mentioned below, also __Running server__ only explains how to start the devserver!

Reply:
Still, people did use this for production, so better to keep the mention here.

Reply:
You can't use something for the productive environment that only applies to the development environment...